cfe_internal/CFE_cfengine.cf

```diff
@@ -31,6 +31,14 @@ bundle agent cfe_internal_management
 
     am_policy_hub.enterprise::
 
+      "hub" -> { "Mission Portal", "CFEngine Enterprise" }
+        usebundle => cfe_internal_purge_scheduled_reports_older_than_days( $(def.purge_scheduled_reports_older_than_days) ),
+        ifvarclass => "enable_cfe_internal_purge_scheduled_reports_older_than_days",
```

Comment:
My reasoning here is that this makes it easier to roll the change across a subset of hosts. Imagine 10k virtual machines sharing the same storage back end all deciding to delete a bunch of old files within the same 5 minutes. Without this enable class, I would have to dig deep into the policy in order to instrument a phased roll out.